endpoint/endpoint.go

```diff
@@ -418,6 +423,19 @@ func (c *ReceiverConfig) copyIn() {
 
 func (c *ReceiverConfig) Validate() error {
 	c.JobID.MustValidate()
```

Comment:
you might have used `MustValidate` below because of this call to `JobID.MustValidate`.

I dont' remember why I made those call sites not just use `JobID.Validate`.

However, a `Validate` function like this one should always error and not panic.